staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -186,7 +186,7 @@ var (
 			Buckets:        compbasemetrics.ExponentialBuckets(1024, 2.0, 8), // 1K, 2K, 4K, 8K, ..., 128K.
 			StabilityLevel: compbasemetrics.ALPHA,
 		},
-		[]string{"group", "version", "kind"},
+		[]string{"group", "version", "resource"},
```

Comment:
@serathius - there are many metrics that you changed that aren't reflected in the changelog

Reply:
I generated the list based on the top comment, this metric is included but with different name `apiserver_watch_events_sizes_count`. 

The `apiserver_` prefix comes from Subsystem and is  the correct part of the metric name. However, the `_count` suffix comes from histogram timeseries name and should be removed.

Let me cross reference it with https://kubernetes.io/docs/reference/instrumentation/metrics/

Reply:
Or, looks like I missed also other metrics. It should be good now.